R/plots.R

```diff
@@ -95,7 +103,11 @@ ggplot_heatmap <- function(xx,
     }
   }
   if (!is.null(custom_hovertext)) {
-    mdf[["text"]] <- paste0(mdf[["text"]], "<br>", custom_hovertext)
+    if (suppress_default_hovertext) {
```

Comment:
All other places you've used ! suppress_default_hovertext, it might be worth be consistent. But your choice on this one.